pandas/tseries/tools.py

```diff
@@ -294,8 +306,16 @@ def to_datetime(arg, errors='raise', dayfirst=False, yearfirst=False,
     >>> %timeit pd.to_datetime(s,infer_datetime_format=False)
     1 loop, best of 3: 471 ms per loop
 
-    """
+    Using non-epoch origins to parse date
+
+    >>> pd.to_datetime([1,2,3], unit='D', origin=Timestamp('1960-01-01'))
+    0    1960-01-01
+    1    1960-01-02
+    ...
+    98   1960-04-08
```

Comment:
The output here (Series of lenght 100) does not correspond to the input (length of 3)